commands/start-container.ts

```diff
@@ -29,7 +29,7 @@ export async function startContainer(context?: ImageNode, interactive?: boolean)
         docker.getExposedPorts(imageToStart.Id).then((ports: string[]) => {
             let options = `--rm ${interactive ? '-it' : '-d'}`;
             if (ports.length) {
-                const portMappings = ports.map((port) => `-p ${port}:${port}`);
+                const portMappings = ports.map((port) => `-p ${port.split("/")[0]}:${port}`);
```

Comment:
Please comment with what you're expecting and the output you're going for.

Reply:
current output: 
`-p 8080:8080`

output I'm going for: 
`-p 8080:8080/tcp` , `-p 3010:3010/tcp`

Notice the difference in 8080/tcp. Earlier, we stripped the protocol when returning the function (see `getExposedPorts`). 

The earlier command was: 
`docker run --rm -d -p 8080:8080 node-todo:latest`
Now with the change: 
`docker run --rm -d -p 8080:8080/tcp node-todo:latest`

Reply:
I wanted code comments. If you need to explain it to me, you need to explain it to code maintainers.

Reply:
Ah, my bad. Added comment